copper-swagger-api/src/main/java/com/pogeyan/swagger/api/utils/SwaggerHelpers.java

```diff
@@ -1227,4 +1251,88 @@ public static String getHostSwaggerUrl() {
 		return hostSwaggerUrl;
 
 	}
+
+	/**
+	 * 
+	 * @param httpRequest
+	 * @return username and password from the request
+	 * @throws UnsupportedEncodingException
+	 */
+
+	private static String getB64Auth(String username, String password) {
+		String source = username + ":" + password;
```

Comment:
Http Utilis reuse getBasic auth method

Reply:
httpUtills method is not there in swagger,so we use this method.